vscode-extension/src/util.ts

```diff
@@ -295,3 +295,19 @@ export function urlJoin(...args) {
   return normalize(parts);
 }
 //#endregion
+
+
+/**
+ * AIConfig Vscode extension has a dependency on the Python extension. 
+ * This function retrieves and returns the path to the current python interpreter.
+ * @returns the path to the current python interpreter
+ */
+export async function getPythonPath(): Promise<string> {
+  const pythonExtension = vscode.extensions.getExtension("ms-python.python");
+  if (!pythonExtension.isActive) {
+    await pythonExtension.activate();
+  }
+  
+  const pythonPath = pythonExtension.exports.settings.getExecutionDetails().execCommand[0];
```

Comment:
If this is null we should default to something. Maybe simply python3 on Mac and py on windows

Reply:
You can check for os and have a getDefaultPython function

Reply:
This is a good idea, will add a diff on top